src/components/OrderHistory/OrderHistoryList.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from "react";
+import { NavLink } from "react-router-dom";
+import { navKeys } from "../../routes/routes";
+
+import { fetchOrderHistory } from "../../common/service/common-service";
+import OrderItem from "./OrderItem";
+
+import { Button } from "../../common/styles/componentsStyles";
+import { StyledOrderHistoryList } from "./OrderHistoryListStyles";
+
+import { OrderInterface } from "../../common/types/common.types";
+
+const OrderHistoryList = () => {
+  const [orders, setOrders] = useState<OrderInterface[] | null>([]);
```

Comment:
in this case we have an initial value of [] so the orders property is never null, and therefore there's no reason to type it as OrderInterface[] | null
```ts
 const [orders, setOrders] = useState<OrderInterface[]>([]);
```